File: Iniciante/1096 - Sequencia IJ 2/Python/1096.py
def main():
    I = 1
    J = 7
    while I <= 9:
        contador = 1
        while contador <= 3:
            print(f"I={I} J={J}")
            J -= 1
            contador += 1
        I += 2
        J = 7
        contador = 1


main()

File: Iniciante/1096 - Sequencia IJ 2/JavaScript/1096.js
function main() {
  let I = 1;
  let J = 7;

  while (I <= 9) {
    let contador = 1;
    while (contador <= 3) {
      console.log(`I=${I} J=${J}`);
      J--;
      contador++;
    }
    I += 2;
    J = 7;
    contador = 1;
  }
}

main();
